doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_architecture_pm.rst

```diff
@@ -250,3 +244,105 @@ Power management benchmark
 **************************
 
 To benchmark the power consumption in *Idle* state, see :ref:`multicore_idle_test`.
+
+Power management optimization example
+*************************************
+
+This example focuses on a HID device use case (for example, a mouse)
+In this use case the typical workflow is the following:
+
+  1. Sample input data from sensors.
+  2. Construct a HID report.
+  3. Transmit the report over the radio.
+  4. Sleep until the next period (usually, less than 1 ms).
+
+The power consumption targets for this example are the following:
+
+* Sleep currents (5 V supply):
+
+  * Application core (Suspend-to-RAM): < 10 µA
+  * Radio core (Suspend-to-Idle, cache disabled): < 10 µA
```

Comment:
This is the mapping between Application states and Software power states:

1. In Application Active state:
    1. Application CPU can be in:
       1. Active
       2. Idle
       3. Suspend-to-RAM
    2. Radio CPU can be in:
       1. Active
       2. Idle
       3. Idle with cache disabled
    3. With this combination of software power states, depending on the current states, the hardware switches between:
       1. System On
       2. System On Idle
       3. System On All Idle

4. In the Idle application state:
   1. Application CPU is in Suspend-to-RAM
   2. Radio CPU is in Idle with cache disabled
   5. With this combination of software power states, the hardware is in System On All Idle